crates/server/primitives/src/admin.rs

```diff
@@ -253,6 +253,33 @@ impl GetContextIdentitiesResponse {
     }
 }
 
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct ContextAlias {
+    pub alias: Alias<PublicKey>,
+    pub identity: PublicKey,
+}
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct ContextAliasesResponseData {
+    pub aliases: Vec<ContextAlias>,
+}
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct GetContextAliasesResponse {
+    pub data: ContextAliasesResponseData,
+}
+
+impl GetContextAliasesResponse {
+    pub const fn new(aliases: Vec<ContextAlias>) -> Self {
+        Self {
+            data: ContextAliasesResponseData { aliases },
+        }
+    }
+}
+
```

Comment:
we won't only be listing identity aliases though, we need to list contexts aliases too

see how we make this accomodation for creating contexts:

https://github.com/calimero-network/core/blob/d7ddaf8562379538647a90e2271194c1d5bf336a/crates/server/primitives/src/admin.rs#L443